golang/cosmos/x/swingset/config.go

```diff
@@ -18,6 +18,9 @@ const DefaultConfigTemplate = `
 [swingset]
 # slogfile is the absolute path at which a SwingSet log "slog" file should be written.
 slogfile = ""
+
+# maxVatsOnline is the maximum number of vats that SwingSet kernel will bring online
+maxVatsOnline = 50
```

Comment:
```suggestion
maxVatsOnline = {{ .Swingset.MaxVatsOnline }}
```

You also need to add a new field to the `SwingsetConfig` type and `DefaultSwingsetConfig` instance, e.g.
```go
type SwingsetConfig struct {
	// SlogFile is the absolute path at which a SwingSet log "slog" file should be written.
	SlogFile string `mapstructure:"slogfile" json:"slogfile,omitempty"`
	// MaxVatsOnline is the maximum number of vats that the SwingSet kernel will have online
	// at any given time.
	MaxVatsOnline int `mapstructure:"maxVatsOnline" json:"maxVatsOnline,omitempty"`
}

var DefaultSwingsetConfig = SwingsetConfig{
	SlogFile: "",
	MaxVatsOnline: 50,
}
```

Reply:
Well, it just happens that I did have a piece of code very similar to it in my local workspace, but since `SlogFile` did not originally have `json:"slogfile,omitempty"` neither did `MaxVatsOnline` and I could not figure out how would it map to `maxVatsOnline`.